lib/coinbase/staking_operation.rb

```diff
@@ -92,6 +92,12 @@ def network
       @network ||= Coinbase::Network.from_id(@model.network_id)
     end
 
+    # Returns the Network ID of the Staking Operation.
+    # @return [Symbol] The Network ID
+    def network_id
+      Coinbase.to_sym(@model.network_id)
+    end
+
```

Comment:
Looks like this already exists above, so can remove `network_id`!
```suggestion
```

Reply:
Done.